procfs/path.go

```diff
@@ -0,0 +1,22 @@
+// Copyright 2016 DigitalOcean
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+// implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package procfs
+
+const defaultProcPath = "/proc"
+
+// ProcPath is the location of /proc. It is exported to allow for overrides
+// to the default value.
+var ProcPath = defaultProcPath
```

Comment:
I tried to figure out any other way. And I went down the path of passing an instance of procfs and found it would take a refactor to do well.